### PR TITLE
Update/org deposit flow

### PR DESCRIPTION
**Pull request by tobi-bams**
## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I have tested on Chrome and Firefox
- [ ] I have tested on a mobile device
- [ ] It has been deployed to people-test.sphinx.chat and tested by others

### CODE REVIEW

**Comment by kevkevinpal** [comment]
can you add screenshots here @tobi-bams

File: frontend/app/src/helpers/helpers.ts [hunk 1]
```diff
@@ -111,7 +111,7 @@ export const sendToRedirect = (url: string) => {
 
 export const calculateTimeLeft = (
```
Comment (kevkevinpal) [inline frontend/app/src/helpers/helpers.ts:112]: we should totally add some sort of unit test for this function

File: frontend/app/src/people/widgetViews/organization/PiadInvoice.tsx
Comment (kevkevinpal) [inline frontend/app/src/people/widgetViews/organization/PiadInvoice.tsx:1]: typo in the name of this file

File: frontend/app/src/people/widgetViews/organization/ExpiredInvoice.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+const ExpiredInvoice = (props: {
+  setLnInvoice: (ln: string) => void;
+  setInvoiceState: (state: InvoiceState) => void;
+}) => {
+  const handleTryAgain = () => {
+    props.setInvoiceState(null);
+    props.setLnInvoice('');
+  };
+  return (
+    <>
```
Comment (kevkevinpal) [inline frontend/app/src/people/widgetViews/organization/ExpiredInvoice.tsx:65]: you dont need these empty div

File: frontend/app/src/people/widgetViews/organization/PiadInvoice.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  font-size: 1.0625rem;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  letter-spacing: 0.01063rem;
+  margin-bottom: 0;
+`;
+
+const PaidInvoice = (props: { amount: number }) => (
+  <>
```
Comment (kevkevinpal) [inline frontend/app/src/people/widgetViews/organization/PiadInvoice.tsx:51]: dont need this empty div

File: frontend/app/src/people/widgetViews/summaries/wantedSummaries/Invoice.tsx [hunk 4]
```diff
@@ -1,45 +1,115 @@
...
+    return () => {
+      if (invoiceTimeout) clearTimeout(invoiceTimeout);
+    };
+  }, [timeLeft, props.invoiceStatus, props]);
 
   return (
-    <div style={{ marginTop: '30px' }}>
-      {timeLeft.seconds >= 0 || timeLeft.minutes >= 0 ? (
+    <>
+      {timeLeft.seconds >= 0 || timeLeft.minutes >= 0 || timeLeft.hours! >= 0 ? (
```
Comment (kevkevinpal) [inline frontend/app/src/people/widgetViews/summaries/wantedSummaries/Invoice.tsx:93]: what is the ! at the end of hours for?

**Review by kevkevinpal — COMMENTED** [review]
looks pretty good to me just added some minor comments

File: frontend/app/src/people/widgetViews/summaries/wantedSummaries/Invoice.tsx (hunk 4 above)
Comment (tobi-bams) [reply frontend/app/src/people/widgetViews/summaries/wantedSummaries/Invoice.tsx]: FIxed..

**Comment by tobi-bams** [comment]
@kevkevinpal and @ecurrencyhodler here are screenshots

Screenshot 2023-11-09 at 21 10 14
Screenshot 2023-11-09 at 21 10 25
Screenshot 2023-11-09 at 21 10 39
Screenshot 2023-11-09 at 21 12 19
Screenshot 2023-11-09 at 21 20 21

File: frontend/app/src/helpers/helpers.ts (hunk 1 above)
Comment (kevkevinpal) [reply frontend/app/src/helpers/helpers.ts:112]: we might need to do this in the future since we don't really have a testing framework in place rn

**Pull request merged**